CodeStyle.Analyzers/CodeStyle.Analyzers.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <IsPackable>true</IsPackable>
+        <RootNamespace>Messerli.CodeStyle.Analyzers</RootNamespace>
+        <LangVersion>8.0</LangVersion>
+        <PackageId>Messerli.CodeStyle.Analyzers</PackageId>
+        <PackageTags>analyzers</PackageTags>
+        <PackageLicenseExpression>MIT OR Apache-2.0</PackageLicenseExpression>
+        <Copyright>© Messerli Informatik AG. All rights reserved.</Copyright>
+        <Description>Various analyzers bundled with opinionated configuration</Description>
+        <NoWarn>$(NoWarn);NU5105;NU5104;NU5128</NoWarn>
```

Comment:
Are these warning suppressions needed?

Reply:
They are not needed. [Removed here](https://github.com/messerli-informatik-ag/code-style/pull/63/commits/5c5e2730e75d903ee0ef4f0fe6c47b6980dfb5e1)